src/test/java/plugins/JobDslPluginTest.java

```diff
@@ -817,6 +822,615 @@ public void should_alert_if_generated_job_changed() {
         assertThat(driver, hasContent(alert));
     }
 
+    /**
+     * Test if a new ListView is created via Job DSL script TextBox.
+     * The created View shall contain a description.
+     */
+    @Test
+    public void is_ListView_created() {
+        String descriptionText = "This is the description of testView";
+        String jobDslScript = "listView('"+ LIST_VIEW_NAME +"') {\n" +
+                "    description('"+descriptionText+"')\n" +
+                "}";
+        openNewlyCreatedListView(jobDslScript, LIST_VIEW_NAME);
+        WebElement description = find(By.xpath("//div[@id='description']/div"));
```

Comment:
The description should be set via script, since it is intended to test this DSL element